### ISSUE TITLE
Incorporate Coda Github Pack Improvements

### ISSUE THREAD

**Issue by jkarpen** [cagov/data-infrastructure#230]
This will involve incorporating new elements that Coda added to the Github pack to allow bringing in Github Milestone data. We are also asking them to add the ability to bring in Project Board data which will let us view certain details like Issue Due Date that get are not available in the Issue API.

**Comment by jkarpen** [comment]
Alan from Coda just added a pack button to allow us to create milestones in Coda and automatically push them to Github so I will be testing that out.

**Comment by jkarpen** [comment]
Worked with Coda to resolve a bug in the Milestone sync that was causing some milestones not to show up in Coda that already existed in Github. 

I have also updated our Milestone table with columns that allow us to create a milestone in Coda, then click a button and have the milestone automatically created in Github. We can also modify things like the title, due date, and description in Coda and have it instantly sync on the Github side. 

There is one issue outstanding: adding milestones one at a time is working, but the functionality to add multiple at the same time with one button-press gave an error during testing. I reached out to Coda support about this. I am going to close this issue and create a new issue to focus on that problem.
